Source/WebCore/dom/make_names.pl

```diff
@@ -247,70 +301,56 @@ sub defaultInterfaceName
 
 ### Parsing handlers
 
-sub valueForName
+sub nameAvoidingCppKeyword
 {
     my $name = shift;
-    my $value = $extensionAttrs{$name};
 
-    if (!$value) {
-        $value = $name;
-        $value =~ s/_/-/g;
-    }
+    my %cppKeywords = map { $_ => 1 } qw(char class default for operator switch template);
```

Comment:
C++ has a lot more keywords than this right? Is this meant to be just those we care about?
Probably a good idea to add a comment explaining why this subset is picked.

Reply:
Yes I just chose the ones that actually conflict with our current set of names. Actually some of those are attributes, and don't need to be there right now. I'll remove those extra ones and add a comment here.